src/applications/disability-benefits/view-payments/containers/App.jsx

```diff
@@ -9,44 +15,61 @@ import {
   thirdSidebarBlock,
 } from '../components/ViewPaymentsSidebar/ViewPaymentsSidebarBlockStates/ViewPaymentsSidebarBlockStates.jsx';
 
-export default function App() {
+function ViewPaymentsApp(props) {
```

Comment:
I think if you rename this here, you may need to update the import and usage here - https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/disability-benefits/view-payments/routes.jsx#L5

Reply:
Ugh, Atom is supposed to catch all of these. Updated